### PR TITLE
DEATH TO WARNINGS!

### PR DESCRIPTION
**Pull request by cjellick**
I couldn't take it anymore.

### CODE REVIEW

File: code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java [hunk 1]
```diff
@@ -304,8 +305,8 @@ public boolean isProjectMember (long projectId, Long usingAccount, boolean isAdm
                 for (ProjectMember member : previousMembers) {
                     otherPreviosMembers.add(new Member(member));
                 }
-                HashSet<Member> create = (HashSet<Member>) ((HashSet<Member>) members).clone();
-                HashSet<Member> delete = (HashSet<Member>) ((HashSet<Member>) otherPreviosMembers).clone();
+                Set<Member> create = new HashSet<Member>(members);
+                Set<Member> delete = new HashSet<Member>(otherPreviosMembers);
```
Comment (cjellick) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java]: @hibooboo2 Was there a specific reason for clone vs the copy constructor here? Just want to make sure I'm not breaking something you did intentionally.

**Comment by sonchang** [comment]
Nice!  They were bugging me too

File: code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubConfigManager.java [hunk 2]
```diff
@@ -57,7 +57,6 @@
     private JsonMapper jsonMapper;
     private ObjectManager objectManager;
     private GithubClient client;
-    private GithubUtils githubUtils;
```
Comment (cjellick) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubConfigManager.java]: @hibooboo2 This is unused. Safe to delete?

**Comment by sonchang** [comment]
LGTM

**Comment by cjellick** [comment]
Want to call out explicitly to all reviewers: I added a checkstyle change that will now cause drone failures on unused imports.

**Comment by alena1108** [comment]
LGTM

**Comment by ibuildthecloud** [comment]
@cjellick I've switched to Intellij which causes problems now.  I never see the warnings anymore.  But I whole heartily support this PR.  (rebase)

**Comment by ibuildthecloud** [comment]
> Want to call out explicitly to all reviewers: I added a checkstyle change that will now cause drone failures on unused imports.

@cjellick Yeah!!!!!!!

**Comment by sonchang** [comment]
Keeping my LGTM :+1:

**Comment by cjellick** [comment]
Rebasing

**Comment by ibuildthecloud** [comment]
@cjellick Need to add a CI check for clean Ecilpse build with no warnings :)

**Comment by cjellick** [comment]
The failed build is a LIE!
http://ci.rancher.io/github.com/rancherio/cattle

...It had a transient failure, I rebuilt, and it is not picking up the new status

File: code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java (hunk 1 above)
Comment (hibooboo2) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java]: There was not a specific reason. This change won't break things.

File: code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubConfigManager.java (hunk 2 above)
Comment (hibooboo2) [inline code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubConfigManager.java]: yes you can remove this. It was used at one point but i stopped using it.

**Comment by hibooboo2** [comment]
LGTM

**Pull request merged**